NOM_visualizations/python/nom_data.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<p>Change "Find the omics processing records" to "Find the data generation records" Also, change the variables from <code>omics_processing_id</code> <code>omicsprocess_has_output</code> <code>omicsprocess_has_input</code> to allude to data generation.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>